action.yml

```diff
@@ -14,7 +16,22 @@ inputs:
     required: false
     default: 'x64'
   jdkFile:
-    description: 'Path to where the compressed JDK is located. The path could be in your source repository or a local path on the agent.'
+    description: 'Path to where the compressed JDK is located. The path could
+       be in your source repository or a local path on the agent.'
+    required: false
+  server-id:
+    description: 'ID of the distributionManagement repository in the pom.xml
+       file.'
+    required: false
+  username:
```

Comment:
Would it be appropriate to call these `maven-username` or `server-username`? I think it makes the YAML read more naturally:

```yml
- name: Set up JDK 1.8
  uses: actions/setup-java@v1
  with:
    java-version: 1.8
    server-id: github
    server-username: ${{ github.actor }}
    server-password: ${{ github.token }}
```

Reply:
I like `server-username` for it's consistency, makes it obvious that all 3 variables are related to each other

Reply:
Going with `server-*`